test-environment.yml

```diff
@@ -30,5 +30,6 @@ dependencies:
   - glue-core>=0.12
   - pip:
     - asteval
+    - git+https://github.com/radio-astro-tools/spectral-cube#spectral-cube
```

Comment:
Shouldn't it be `git+https://github.com/radio-astro-tools/spectral-cube#egg=spectral-cube`? (the `egg` part)
If not, it looks good to me.

Reply:
Yes, you're right! It was just a typo but it looks like it worked anyway. Will fix.